cotainr/__init__.py

```diff
@@ -7,6 +7,8 @@
 
 """
 
+import sys
+
 __version__ = "2023.02.0"
```

Comment:
Are we waiting for the output change to create a new release?

Reply:
Currently, we have #33, #34, and #36 staged for release when approved and merged. None of those seem critical to release ASAP, so I suggest we at least include any needed fix for #35 and possibly also a reworked version of #32 before making a new release.